src/updates/node/installer.ts

```diff
@@ -0,0 +1,38 @@
+import { exec } from 'child_process';
+import * as semver from 'semver';
+
+export function checkNodeInstalled(): boolean {
+	try {
+		exec('node -v');
+	} catch (e) {
+		return false;
+	}
+	return true;
+}
+
+export function checkNodeVersion(): string {
```

Comment:
I think this should be combined with the existing `checkNodeInstalled` on line 4 to become the `checkInstalledVersion` function